src/pages/index.js

```diff
@@ -59,55 +35,175 @@ function renderCard(cardData) {
   cardSection.addItem(element);
 }
 
-const cardSection = new Section(
-  { items: initialCards, renderer: renderCard },
-  ".cards__list"
+// INITIAL CARDS AND USER INFO //
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "67f9ee29-81ba-42c1-865a-539d34535736",
+    "Content-Type": "application/json",
+  },
+});
+
+let cardSection;
+
+api.loadPageContent().then(([cards, userData]) => {
+  cardSection = new Section(
+    {
+      items: cards,
+      renderer: renderCard,
+    },
+    ".cards__list"
+  );
+  cardSection.renderItems();
+
+  newUserInfo.setUserAvatar(userData.avatar);
+  newUserInfo.setUserInfo({
+    name: userData["name"],
+    about: userData["about"],
+    id: userData["id"],
+  });
+});
```

Comment:
Needs correction: don't forget to handle api errors (add a `catch` block for every api call)

Reply:
fixed, and changed order to remove console error